Cognite.Simulator.Tests/UtilsTests/SimulationRunnerTest.cs

```diff
@@ -388,6 +388,12 @@ public class SampleSimulationRunner :
         {
         }
 
+        protected override async Task EndSimulationRun(SimulationRunEvent simEv,
+            CancellationToken token)
+        {
+
```

Comment:
```suggestion
await Task.CompletedTask;
```
I haven't tried this but might help suppressing the warning